### PR TITLE
docs: dasharo_tools_suite.md: prepare for dts rel v1.0.0

### PR DESCRIPTION
**Pull request by TomaszAIR**
Signed-off-by: Tomasz Żyjewski <tomasz.zyjewski@3mdeb.com>

### CODE REVIEW

**Comment by pietrushnic** [comment]
- [ ] On top of the page (introduction or before) could be DTS start screen
- [ ] There are no release notes nor information, about supported hardware
- [ ] `DTS CE functionality` should enumerate CE features
- [ ] DTS should be at top level in menu
- [ ] There is no information how to provide feedback/submit bugs

File: docs/common-coreboot-docs/dasharo_tools_suite.md [hunk 1]
```diff
@@ -29,42 +21,42 @@ Currently, there are:
...
-This documentation is compatible with the `v0.3.0` version of the DTS.
+This section describes how to boot DTS using USB stick.
 
 ### Requirements
 
 * USB stick (at least 1GB)
 * Wired network connection
 * Secure Boot disabled
-* [DTS CE v0.3.0 downloaded](https://cloud.3mdeb.com/index.php/s/Q7pAppm4gnRCef9/download)
+* [DTS CE v1.0.0 downloaded](https://cloud.3mdeb.com/index.php/s/aB6dCdKLB33oZmC)
```
Comment (pietrushnic) [inline docs/common-coreboot-docs/dasharo_tools_suite.md:41]: Why we replacing? Are you sure v0.3.0 would not be needed anyome?

File: docs/common-coreboot-docs/dasharo_tools_suite.md [hunk 2]
```diff
@@ -76,20 +68,33 @@ This documentation is compatible with the `v0.1.0` version of the DTS.
...
 
-### Using DTS
+## Dasharo firmware update
 
 Within DTS, you may use the `flashrom` and `fwupdmgr` utilities to update,
 downgrade, or reinstall your firmware.
 
-To update your firmware to the latest version:
+To update your firmware to the latest version first choose option number 9 to
+drop to shell a next run:
```
Comment (pietrushnic) [inline docs/common-coreboot-docs/dasharo_tools_suite.md:97]:
```suggestion
drop to Shell and run:
```

Comment (TomaszAIR) [reply docs/common-coreboot-docs/dasharo_tools_suite.md]: Fixed https://github.com/Dasharo/docs/pull/260/commits/77399300de37af7ab3b2de0ece625cd1d7dc1ea4#diff-b5545f47decb8260688b188363790fc5aef4ea727b9c363329f070d1fa3664a4R129

**Review by pietrushnic — APPROVED** [review]

**Pull request merged**